.github/pr-commands.json

```diff
@@ -165,5 +165,11 @@
     "matches": [ "public/app/plugins/datasource/zipkin/**/*"],
     "action": "updateLabel",
     "addLabel": "datasource/Zipkin"
+  },
+  {
+    "type": "changedfiles",
+    "matches": ["public/app/features/variables/**/*", "public/app/features/templating/**/*"],
+    "action": "updateLabel",
+    "addLabel": "area/dashboard/templating"
   }
```

Comment:
Will this replace the `area/frontend` label or add `area/dashboard/templating` label @marefr?

Reply:
good question, can multiple rules match or just one?

Reply:
if I read this test, correctly it will just add the label https://github.com/grafana/github-actions-testrepo/pull/18

Reply:
It will add area/dashboard/templating

Reply:
And area/frontend for example